cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java

```diff
@@ -100,6 +102,32 @@ public Properties getExtensionProperties() {
       }
 
       @Override
+      public void createTopic(String s) throws TopicAlreadyExistsException, IOException {
```

Comment:
Just a thought... isn't it weird to have such methods in a `AuthorizationContext` class?